boat_simulator/nodes/physics_engine/force_computation.py

```diff
@@ -0,0 +1,43 @@
+"""Class to compute lift and drag forces acting on a medium."""
+
+from typing import Tuple
+
+import numpy as np
+from numpy.typing import NDArray
+
+from boat_simulator.common.types import Scalar
+
+
+class ForceComputation:
+    def __init__(self, lift_coefficient: Scalar, area: Scalar, fluid_density: Scalar):
+        self.__lift_coefficient = lift_coefficient
+        self.__area = area
+        self.__fluid_density = fluid_density
```

Comment:
Both the area and lift coefficient are functions of the angle of attack. For the purposes of this class, we care more about the lookup tables that we would use to compute such values, so storing a specific lift coefficient and area is not sufficient.